mlir/lib/Dialect/Utils/ReshapeOpsUtils.cpp

```diff
@@ -483,3 +483,13 @@ PackingMetadata mlir::computePackingMetadata(int64_t packedRank,
   }
   return res;
 }
+
+OpFoldResult mlir::reshapeConstantSource(DenseElementsAttr source,
```

Comment:
Is the comment not relevant any more?

Reply:
Note that I create a declaration in this header file:
* mlir/include/mlir/Dialect/Utils/ReshapeOpsUtils.h

That's where the comment got relocated :)